src/Compilation.zig

```diff
@@ -725,6 +730,7 @@ pub const InitOptions = struct {
     test_filter: ?[]const u8 = null,
     test_name_prefix: ?[]const u8 = null,
     subsystem: ?std.Target.SubSystem = null,
+    wasi_exec_model: ?WasiExecModel = null,
```

Comment:
Could you add a sentence or two explaining what this option holds?

Reply:
https://github.com/ziglang/zig/pull/8978/commits/cd610975f2c401ec413787c04c2c1c8a34e54804 fixed!